exercise/dict_has_duplicates.py

```diff
@@ -0,0 +1,28 @@
+"""
+This module contains a code for exercises 11-4 related to:
+Think Python, 2nd Edition
+Chapter 11: Dictionaries
+
+Takes a list as a parameter and returns True if there is any object that appears more
+than once in the list.
+"""
+def has_duplicates(list_input):
+    """
+    Check if there are any duplicate elements
+
+    Input: list
```

Comment:
Invalid format, instead `Input: list` we can use
```
Parameter:
    - list_input: list
 ```
 Or
```
list_input: list
```

Please double-check and apply for other files.